src/plugins/controls/public/options_list/components/options_list_editor_options.tsx

```diff
@@ -39,16 +42,41 @@ export const OptionsListEditorOptions = ({
           }}
         />
       </EuiFormRow>
-      <EuiFormRow>
+      <EuiFormRow display="columnCompressedSwitch">
         <EuiSwitch
-          label={OptionsListStrings.editor.getRunPastTimeoutTitle()}
-          checked={Boolean(state.runPastTimeout)}
+          label={OptionsListStrings.editor.getAllowExclude()}
+          checked={!state.allowExclude}
           onChange={() => {
-            onChange({ runPastTimeout: !state.runPastTimeout });
-            setState((s) => ({ ...s, runPastTimeout: !s.runPastTimeout }));
+            onChange({ allowExclude: !state.allowExclude });
+            setState((s) => ({ ...s, allowExclude: !s.allowExclude }));
           }}
         />
```

Comment:
This adds a setting for the options list that allows authors to disable the option of allowing users to exclude their selections.

https://user-images.githubusercontent.com/8698078/197568216-2a4a5865-edfe-4067-8371-5f15bfcfda4f.mov

@elastic/kibana-docs The current wording of this is "Allow selections to be excluded" for this setting - would appreciate your thoughts! 👍

Reply:
Wording LGTM!